.github/ISSUE_TEMPLATE/bug_report_cli.yml

```
@@ -1,14 +1,49 @@
 name: "\U0001F41B CLI Bug Report"
-description: 'You want to report an issue with Expo CLI, our command line tool'
-labels: ['stale', 'pending closure']
+description: 'Report a reproducible bug in the versioned Expo CLI (npx expo)'
+labels: ['needs review', 'CLI']
 body:
   - type: markdown
     attributes:
-      value: This is not the repository for Expo CLI. Please post this issue in the https://github.com/expo/expo-cli repo instead.
+      value: Thanks for taking the time to file a bug report! Please fill out this form as completely as possible.
+  - type: markdown
+    attributes:
+      value: If you leave out sections there is a high likelihood your issue will be closed. If you have a question, not a bug report, please post it on our [forums](https://forums.expo.dev/) instead.
+  - type: textarea
+    attributes:
+      label: Summary
+      description: Describe the issue in 1 or 2 sentences
+      placeholder: Clearly describe what the expected behavior is vs. what is actually happening. This should be as short as possible, while still communicating all the necessary information. If your summary is simply 'my device cannot connect to wifi', then you need to continue debugging yourself and provide more information.
```

```suggestion
      placeholder: Clearly describe what the expected behavior is vs. what is actually happening. This should be as short as possible, while still communicating all the necessary information. If your summary is simply, for example: 'My device cannot connect to Wi-Fi.', then you need to continue debugging yourself and provide more information.
```

Small clarification that the sentence is just an example.